search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -714,6 +715,19 @@
                 (format "Cannot set relevance boost on field [%s]." (csk/->snake_case_string field)))))
           (seq boosts))))
 
```

Comment:
Do you think we should add another validation for shapefile size?

I know we have some middleware validation that no request body can be more than 50?(configurable) MB. Do we need a more strict size check for shapefiles?

Reply:
There is a separate ticket for this: CMR-6205